client/app/private-database/configuration/private-database-configuration.html

```diff
@@ -1,76 +1,60 @@
-                    data-ng-click="configurationsCtrl.edit.value = true"
-                    data-ng-disabled="database.expired || isRebooting"
-                    data-ng-if="!configurationsCtrl.edit.value">
-            </button>
-            <button class="btn btn-block btn-primary" type="button"
-                    data-i18n-static="privateDatabase_configuration_confirm_btn"
-                    data-ng-click="configurationsCtrl.updateConfigurations()"
-                    data-ng-disabled="database.expired"
-                    data-ng-if="configurationsCtrl.edit.value">
-            </button>
-            <button class="btn btn-block btn-default" type="button"
-                    data-i18n-static="privateDatabase_configuration_cancel_btn"
-                    data-ng-click="configurationsCtrl.configurations = configurationsCtrl.baseConfiguration; configurationsCtrl.edit.value = false"
-                    data-ng-disabled="database.expired"
-                    data-ng-if="configurationsCtrl.edit.value">
-            </button>
+            <form name="dbConfigurationForm" data-ng-submit="configurationsCtrl.updateConfigurations()">
+                <div data-ng-repeat="configuration in configurationsCtrl.configurations track by $index">
+                    <oui-field label="{{configuration.key}}"
+                               size="m"
+                               ng-if="configuration.type === 'select'"
+                               help-text="{{configuration.description}}">
+                        <oui-select id="{{configuration.key + 'Field'}}"
+                                    name="{{configuration.key + 'Field'}}"
+                                    data-title="Select a letter"
+                                    model="configuration.selectedValue"
+                                    items="configuration.availableValues"
+                                    match="text"
+                                    required
+                                    data-align="start">
+                            <span ng-bind="$item.text"></span>
+                        </oui-select>
+                    </oui-field>
+                    <oui-field label="{{configuration.key}}"
+                               size="s"
+                               ng-if="configuration.type === 'toggle'"
+                               help-text="{{configuration.description}}">
+                        <oui-radio-toggle-group model="configuration.selectedValue.id">
+                           <oui-radio ng-repeat="item in configuration.availableValues"
+                                      text="{{item.text}}"
+                                      value="item.id"></oui-radio>
+                        </oui-radio-toggle-group>
+                    </oui-field>
+                </div>
+                <oui-form-actions submit-text="{{ tr('privateDatabase_configuration_confirm_btn') }}"></oui-form-actions>
```

Comment:
Use one-time binding for translations and everywhere it's possible
data- for submit-text